packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -580,3 +580,110 @@ class _UiKitPlatformView extends LeafRenderObjectWidget {
     renderObject.updateGestureRecognizers(gestureRecognizers);
   }
 }
+
+/// Embeds an existing platform view in the widget tree.
+///
+/// A "PlatformView" is a view that being displayed with the help of the platform's UI system
+/// rather than completely being painted by the Flutter Engine. For example, the webview_flutter
+/// plugin used a pre-made "PlatformView" -- such as "AndroidView" for Android and "UIKitView" for iOS
```

Comment:
not sure what do you mean by pre-made platform view?

Reply:
by pre-made platform view, I meant the platformviews that are pre-made in the framework, like "AndroidView" or "UIKitView"

Reply:
I find it a little confusing here.